src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,11 @@
-const booleanToWord = boolean => {};
+const booleanToWord = boolean => {
+    const a = boolean.toString();
```

Comment:
There is no need to convert the boolean to a string here. Booleans are one of two values, `true` or `false`. `true` is truthy and `false` is falsy. Therefore you can do something as simple as checking the truthiness of the `boolean` parameter and return the strings using a ternary expression. eg `return boolean ? 'Yes' : 'No'`. Or if you still wanted to use if statements, you could refactor to `if (boolean) { 
  return 'Yes'; 
} 
return 'No';`

Reply:
You don't need a second if statement if there are only two possible options to check. For example with booleans, we know it will be true, or false, a boolean cannot be any other value. Therefore if you check whether something is true, all the true statements will hit the first return. Anything that is not true, will be false. Therefore anything that fails the first if statement will be false, so we can just `return 'No';`

Reply:
When you are naming variables, you want to name them something descriptive too. So `a` can be improved to something like `booleanString` as the value stored inside is a data type of string that will be "true" or "false".